var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -109,6 +109,7 @@ class Mvapich2(AutotoolsPackage):
     )
 
     variant("hwlocv2", default=False, description="Builds mvapich2 with hwloc v2")
+    variant("hwloc-graphics", default=False, description="Enables hwloc graphics")
```

Comment:
Can we do:
```suggestion
    variant("hwloc_graphics", default=False, description="Enables hwloc graphics")
```
for consistency with other variants?